specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/rbac.json

```diff
@@ -0,0 +1,610 @@
+            "name": "RoleDefinitionType",
+            "modelAsString": false
+          }
+        },
+        "assignableScopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A set of fully qualified Scopes at or below which Role Assignments may be created using this Role Definition. This will allow application of this Role Definition on the entire database account or any underlying Database / Collection. Must have at least one element. Scopes higher than Database account are not enforceable as assignable Scopes. Note that resources referenced in assignable Scopes need not exist."
+        },
+        "permissions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Permission"
+          },
+          "description": "The set of operations allowed through this Role Definition."
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "Properties to update or create an Azure Cosmos DB Role Definition."
+    },
+    "SqlRoleDefinitionGetResults": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified GUID of the Role Definition containing the resource path and the ID supplied for this Role Definition during create."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The ID supplied for this Role Definition during create."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of resource."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SqlRoleDefinitionCreateUpdateParameters",
+          "description": "Properties related to the Role Definition."
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "An Azure Cosmos DB Role Definition."
+    },
+    "Permission": {
+      "type": "object",
+      "properties": {
+        "dataActions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An array of data actions that are allowed."
+        }
+      },
+      "description": "The set of data plane operations permitted through this Role Definition."
+    },
+    "DefaultErrorResponse": {
+      "properties": {
+        "error": {
+          "properties": {
+            "code": {
+              "description": "Error code.",
+              "type": "string"
+            },
+            "message": {
+              "description": "Error message indicating why the operation failed.",
+              "type": "string"
+            }
+          }
+        }
+      },
+      "description": "An error response from the service."
+    },
+    "SqlRoleDefinitionListResult": {
+      "properties": {
+        "value": {
```

Comment:
add "readOnly": true as a key-value pair inside "value", same for SqlRoleAssignmentListResult object.

Reply:
Done